src/main/clojure/clara/tools/testing_utils.cljc

```diff
@@ -62,14 +62,15 @@
                                                                                                        cat
                                                                                                        session-opts)))]))
                                                         cat)))
-           
+
            test-form `(~(if (com/compiling-cljs?)
```

Comment:
I almost commented here that this `if` case is unnecessary since in [CLJS v1.9.198](https://github.com/clojure/clojurescript/blob/master/changes.md#19198) you can refer to namespaces via `clojure.*` that do not exists, but do exists in `cljs.*` and it will automatically resolve it. This was done in https://dev.clojure.org/jira/browse/CLJS-1692

However, then I realized that Clara is building against a really old CLJS now (by CLJS standards). No action to take here and I'm not saying we should move all the way up to a 1.9.198 minimum. However, I did log #363 when I realized this situation.